tests/sentry/api/endpoints/test_oauth_userinfo.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+from sentry.models import ApiToken
+from sentry.testutils import APITestCase
+from sentry.testutils.silo import control_silo_test
+
+
+@control_silo_test(stable=True)
+class OAuthUserInfoTest(APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.login_as(self.user)
+        self.path = reverse(
+            "sentry-api-0-oauth-userinfo",
+        )
+        self.client = APIClient()
+
+    def test_requires_access_token(self):
+        response = self.client.get(self.path)
+
+        assert response.status_code == 401
+        assert response.data == "No access token found."
```

Comment:
Generally our error responses are JSON and have a `detail` key with the message in it.

Reply:
Fixed!